src/xdocs/checks.xml

```diff
@@ -364,6 +364,12 @@
           Checks that interface type parameter names conform to a format specified
         by the format property.</td>
       </tr>
+      <tr>
+        <td><a href="config_naming.html#InvalidJavadocPosition">
+          InvalidJavadocPosition</a></td>
+        <td>
+          Checks that Javadocs are located at the correct position.</td>
```

Comment:
this summary kind of prove that we need to drop "Invalid" from name of Check.

Reply:
I don't see how, `correct` is just the negative of `invalid`.

Reply:
it was a one more hint that something wrong with name.
let skip this, as we do not have better name, lets use what we have.